### PR TITLE
chore: remove clear text token from Client Libraries code sample blocks 

### LINKED ISSUE #3655: Client libraries: Remove clear-text token
We are changing the UI so that you can no longer see the text of a token after it has been created.  The load data/client library pages currently show the clear text token of the selected token.  We need to remove this functionality - maybe we can copy what we have done on the Telegraf config pages, where we just reference the INFLUX_TOKEN environment variable, and let people know to set that variable for their client library use.

### PR DESCRIPTION
**Pull request by blegesse-git**
Closes #3655 

PR removes: 
1. Token functionality from the Client Libraries `Code Sample Options`. 
-  The plan is to have users create tokens from the API Token page of UI 
2. Removes clear text tokens from `Initialize the Client` code sample block. 
- working on adding code to reference environmental variable in each language
- remaining CLs: Swift, R, C#, and Arduino

Client libraries page before PR: 
Screen Shot 2022-01-26 at 10 38 42 AM
Client libraries page after PR: 
Screen Shot 2022-01-26 at 10 48 41 AM

### CODE REVIEW

File: src/writeData/clients/Javascript/initialize.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const {InfluxDB} = require('@influxdata/influxdb-client')
 
 // You can generate an API token from the "API Tokens Tab" in the UI
-const token = '<%= token %>'
+const token = '<INFLUX_TOKEN>'
```
Comment (sranka) [inline src/writeData/clients/Javascript/initialize.example:4]: to get the token from the system environment in node.js:
```suggestion
const token = process.env.INFLUX_TOKEN
```
FYI: The javascript client library also can also run in a browser or in Deno.js . The browser cannot get system environment vars, Deno.js would use `Deno.env.get('INFLUX_TOKEN')`

File: src/writeData/clients/Java/initialize.example [hunk 2]
```diff
@@ -16,7 +16,7 @@ public class InfluxDB2Example {
   public static void main(final String[] args) {
 
     // You can generate an API token from the "API Tokens Tab" in the UI
-    String token = "<%= token %>";
+    String token = "<INFLUX_TOKEN>";
```
Comment (rhajek) [inline src/writeData/clients/Java/initialize.example:19]:
```suggestion
    String token = System.getenv("INFLUX_TOKEN");
```

File: src/writeData/clients/Kotlin/initialize.example [hunk 3]
```diff
@@ -13,7 +13,7 @@ import java.time.Instant
 fun main() = runBlocking {
 
     // You can generate an API token from the "API Tokens Tab" in the UI
-    val token = "<%= token %>"
+    val token = "<INFLUX_TOKEN>"
```
Comment (rhajek) [inline src/writeData/clients/Kotlin/initialize.example:16]:
```suggestion
    val token = System.getenv()["INFLUX_TOKEN"]
```

File: src/writeData/clients/PHP/initialize.example [hunk 4]
```diff
@@ -3,7 +3,7 @@ use InfluxDB2\Model\WritePrecision;
 use InfluxDB2\Point;
 
 # You can generate an API token from the "API Tokens Tab" in the UI
-$token = '<%= token %>';
+$token = '<INFLUX_TOKEN>';
```
Comment (rhajek) [inline src/writeData/clients/PHP/initialize.example:6]:
```suggestion
$token = getenv('INFLUX_TOKEN');
```

File: src/writeData/clients/Scala/initialize.example [hunk 5]
```diff
@@ -15,7 +15,7 @@ object InfluxDB2ScalaExample {
   def main(args: Array[String]): Unit = {
 
     // You can generate an API token from the "API Tokens Tab" in the UI
-    val token = "<%= token %>"
+    val token = "<INFLUX_TOKEN>"
```
Comment (rhajek) [inline src/writeData/clients/Scala/initialize.example:18]:
```suggestion
    val token = sys.env.get("INFLUX_TOKEN")
```

**Review by vlastahajek — CHANGES_REQUESTED** [review]
Go client complete change of `src/writeData/clients/Go/initialize.example`:
```go
package main

import (
  "context"
  "fmt"
  "os"
  "time"

   "github.com/influxdata/influxdb-client-go/v2"
)

func main() {
  // Create a client
  // You can generate an API Token from the "API Tokens Tab" in the UI
  client := influxdb2.NewClient("<%= server %>", os.Getenv("INFLUX_TOKEN"))
  // always close client at the end
  defer client.Close()
}

```

File: src/writeData/clients/Python/initialize.example [hunk 6]
```diff
@@ -4,7 +4,7 @@ from influxdb_client import InfluxDBClient, Point, WritePrecision
 from influxdb_client.client.write_api import SYNCHRONOUS
 
 # You can generate an API token from the "API Tokens Tab" in the UI
-token = "<%= token %>"
+token = "<INFLUX_TOKEN>"
```
Comment (rhajek) [inline src/writeData/clients/Python/initialize.example:7]:
```suggestion
token = os.getenv("INFLUX_TOKEN")
```

Comment (rhajek) [reply src/writeData/clients/Python/initialize.example:7]: It is also needed to add an import at the start of example:

```python
import os
```

File: src/writeData/clients/Ruby/initialize.example [hunk 7]
```diff
@@ -1,7 +1,7 @@
 require 'influxdb-client'
 
 # You can generate an API token from the "API Tokens Tab" in the UI
-token = '<%= token %>'
+token = '<INFLUX_TOKEN>'
```
Comment (rhajek) [inline src/writeData/clients/Ruby/initialize.example:4]:
```suggestion
token = ENV["INFLUX_TOKEN"]
```

File: src/writeData/clients/Python/initialize.example (hunk 6 above)
Comment (blegesse-git) [reply src/writeData/clients/Python/initialize.example]: done!

**Review by vlastahajek — APPROVED** [review]
👍

**Review by sranka — APPROVED** [review]

File: src/writeData/clients/Arduino/initialize.example [hunk 8]
```diff
@@ -18,7 +18,7 @@ ESP8266WiFiMulti wifiMulti;
 // InfluxDB v2 server url, e.g. https://eu-central-1-1.aws.cloud2.influxdata.com (Use: InfluxDB UI -> Load Data -> Client Libraries)
 #define INFLUXDB_URL "<%= server %>"
 // InfluxDB v2 server or cloud API token (Use: InfluxDB UI -> Data -> API Tokens -> <select token>)
```
Comment (vlastahajek) [inline src/writeData/clients/Arduino/initialize.example:20]:
```suggestion
// InfluxDB v2 server or cloud API token (Use: InfluxDB UI -> Data -> API Tokens -> Generate API Token)
```

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**